<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClipPackageDialog</class>
 <widget class="QDialog" name="ClipPackageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Select Clip Package</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="errorLabel">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Game Mode:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="gameModesComboBox">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Difficulty:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="difficultyComboBox">
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
        <item>
         <property name="text">
          <string>Easy</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hard</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mahlerisch</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="advancedGroupBox">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <pointsize>25</pointsize>
         </font>
        </property>
        <property name="title">
         <string>Advanced</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QWidget" name="advancedWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>25</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Clip Packages:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="clipPackagesWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="useMaxRoundsCheckBox">
              <property name="font">
               <font>
                <pointsize>25</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Use Max Rounds</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="maxRoundsSpinBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="font">
               <font>
                <pointsize>25</pointsize>
               </font>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="cancelPushButton">
          <property name="text">
           <string>Cancel</string>
          </property>
          <property name="icon">
           <iconset resource="../oxygen-icons-png-master/oxygen_windows.qrc">
            <normaloff>:/themes/oxygen/32x32/actions/dialog-close.png</normaloff>:/themes/oxygen/32x32/actions/dialog-close.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="okPushButton">
          <property name="text">
           <string>OK</string>
          </property>
          <property name="icon">
           <iconset resource="../oxygen-icons-png-master/oxygen_windows.qrc">
            <normaloff>:/themes/oxygen/32x32/actions/dialog-ok-apply.png</normaloff>:/themes/oxygen/32x32/actions/dialog-ok-apply.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../oxygen-icons-png-master/oxygen_windows.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>useMaxRoundsCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>maxRoundsSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>426</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>478</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedGroupBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>advancedWidget</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>130</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>307</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
